.github/workflows/docker.yml

```diff
@@ -21,19 +21,30 @@ jobs:
     - name: Checkout
       uses: actions/checkout@v3
 
+    - name: Extract metadata (tags, labels) for Docker
+      id: meta
+      uses: docker/metadata-action@818d4b7b91585d195f67373fd9cb0332e31a7175
+      with:
+        images: ${{ env.REGISTRY }}/cosmos/${{ env.IMAGE_NAME }}
+
+    - name: Build and push Docker image
+      uses: docker/build-push-action@2eb1c1961a95fc15694676618e422e8ba1d63825
+      with:
+        context: .
+        tags: ${{ steps.meta.outputs.tags }}
+        build-args: |
```

Comment:
if neither `push` or `load` are specified, what exactly does this do?

Reply:
just the build. Unsure if there's a better way to do this (combining this step with `test simd is runnable`?)

Reply:
ah I guess it doesn't export it to docker? As far as I could tell, this did use the correct image [with the run I triggered on my fork](https://github.com/DimitrisJim/ibc-go/actions/runs/5296535670/jobs/9587707581#step:5:1). Lmk if I'm missing something here.